File: M3Tools/Dialogs/User/LoginDialog.vb
Imports System.Windows.Forms

Public Class LoginDialog
	''' <summary>
	''' Gets or sets the username
	''' </summary>
	''' <returns></returns>
	Public Property Username As String
		Get
			Return LoginFields1.Username
		End Get
		Set(value As String)
			LoginFields1.Username = value
		End Set
	End Property

	''' <summary>
	''' Gets or sets the password
	''' </summary>
	''' <returns></returns>
	Public Property Password As String
		Get
			Return LoginFields1.Password
		End Get
		Set(value As String)
			LoginFields1.Password = value
		End Set
	End Property

	Private Sub SubmitCredentials(sender As Object, e As EventArgs) Handles btn_Submit.Click
		Me.DialogResult = DialogResult.OK
		Me.Close()
	End Sub

	Private Sub CancelRequest(sender As Object, e As EventArgs) Handles btn_Cancel.Click
		Me.DialogResult = DialogResult.Cancel
		Me.Close()
	End Sub
End Class

File: M3Tools/Dialogs/User/LoginDialog.cs
using System;
using System.Windows.Forms;

namespace SPPBC.M3Tools
{

    public partial class LoginDialog
    {
        /// <summary>
	/// 	''' Gets or sets the username
	/// 	''' </summary>
	/// 	''' <returns></returns>
        public string Username
        {
            get
            {
                return LoginFields1.Username;
            }
            set
            {
                LoginFields1.Username = value;
            }
        }

        /// <summary>
	/// 	''' Gets or sets the password
	/// 	''' </summary>
	/// 	''' <returns></returns>
        public string Password
        {
            get
            {
                return LoginFields1.Password;
            }
            set
            {
                LoginFields1.Password = value;
            }
        }

		/// <summary>
		/// 
		/// </summary>
        public LoginDialog()
        {
            InitializeComponent();
        }

        private void SubmitCredentials(object sender, EventArgs e)
        {
            DialogResult = DialogResult.OK;
            Close();
        }

        private void CancelRequest(object sender, EventArgs e)
        {
            DialogResult = DialogResult.Cancel;
            Close();
        }
    }
}
